src/kbmod/search/common.h

```diff
@@ -111,6 +113,34 @@ struct trajRegion {
     float flux;
 };
 
+struct stampParameters {
+    int radius = 10;
+    StampType stamp_type = STAMP_SUM;
+    bool do_filtering = false;
```

Comment:
For some reason, I'm only just now noticing lots of snake_case naming has been added to the c++ code. Should there be a preferred convention going forward?

Reply:
Good point. In general we have been moving the code toward LSST DM's style guide (https://developer.lsst.io/cpp/style.html#variable-names-must-be-camelcase-or-snake-case), but that supports both camelcase and snakecase. I am fine with either, but tend to default to snake case. I would be interested to hear if there is a general preference.